app/src/main/java/org/xwiki/android/sync/bean/UserPayload.kt

```diff
@@ -53,7 +53,7 @@ class UserPayload : Parcelable {
         dest.writeString(this.phone)
     }
 
-    constructor() {}
+    constructor()
```

Comment:
This constructor was created for compatibility with Java? If not - what for?

Reply:
Removed, as it was not used anywhere.